### ISSUE TITLE
Add the Meilisearch cloud-hosted solution

### ISSUE THREAD

**Issue by alallema** [meilisearch/firestore-meilisearch#99]
For now, the `Meilisearch` cloud-hosted solution is still in beta but we should think about adding it to the prerequisites of the `PREINSTALL.md` file as a possibility to create a ready-made `Meilisearch` instance.

**Comment by bidoubiwa** [comment]
In the pre-install we redirect to the documentation which suggest using the cloud. Are we sure we want to duplicate this information?

**Comment by alallema** [comment]
>In the pre-install we redirect to the documentation which suggest using the cloud. Are we sure we want to duplicate this information?

For now, in the PREINSTALL we just refer to the documentation to run an instance of `Meilisearch`, it might be a good point to redirect the user directly to the `Meilisearch` cloud no?
```Run a Meilisearch instance. There are many easy ways [to download and run a Meilisearch instance](https://docs.meilisearch.com/learn/getting_started/installation.html#download-and-launch)```

**Comment by bidoubiwa** [comment]
Would you say both or only the cloud?

Personally, I prefer avoiding linking to both the doc and the cloud. We avoid having to maintain linking and wording that the doc already does on that same link.

**Comment by alallema** [comment]
> Personally, I prefer avoiding linking to both the doc and the cloud. We avoid having to maintain linking and wording that the doc already does on that same link.

Yes, I agree but I also think that the user who used firebase will be more likely to prefer to use the cloud no? And I like the fact that the users can have all the options ... Like your prefer
